extensions/resource-deployment/src/services/tools/toolBase.ts

```diff
@@ -100,8 +100,8 @@ export abstract class ToolBase implements ITool {
 		return <string>toolStatusLocalized.get(this._status);
 	}
 
-	public get autoInstallRequired(): boolean {
-		return this.status !== ToolStatus.Installed && this.autoInstallSupported;
+	public isInstallRequired(minimumVersion?: string): boolean {
```

Comment:
>isInstallRequired [](start = 8, length = 17)

I'd suggest renaming this to canAutoInstall or something similar - this isn't really saying anything about the install being "required" in my eyes. Just that we can install a valid version.

Reply:
will rename to isMissingOrOld